lib/screen/account/persona_details_page.dart

```diff
@@ -204,163 +215,185 @@ class _PersonaDetailsPageState extends State<PersonaDetailsPage>
-            .pushNamed(AppRouter.personaConnectionsPage, arguments: payload);
+            .pushNamed(AppRouter.walletDetailsPage, arguments: payload);
       },
     );
   }
 
   Widget _preferencesSection() {
     final theme = Theme.of(context);
-    return Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
-      Text(
-        "preferences".tr(),
-        style: theme.textTheme.headline1,
-      ),
-      const SizedBox(
-        height: 14,
-      ),
-      Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: [
-              Text('hide_from_collection'.tr(),
-                  style: theme.textTheme.headline4),
-              CupertinoSwitch(
-                value: isHideGalleryEnabled,
-                onChanged: (value) async {
-                  await injector<AccountService>()
-                      .setHidePersonaInGallery(widget.persona.uuid, value);
-                  final hiddenAddress =
-                      await injector<AccountService>().getHiddenAddresses();
-                  setState(() {
-                    context
-                        .read<NftCollectionBloc>()
-                        .add(UpdateHiddenTokens(ownerAddresses: hiddenAddress));
-                    isHideGalleryEnabled = value;
-                  });
-                },
-                activeColor: theme.colorScheme.primary,
-              )
-            ],
-          ),
-          const SizedBox(height: 14),
-          Text(
-            "do_not_show_nft".tr(),
-            //"Do not show this account's NFTs in the collection view.",
-            style: theme.textTheme.bodyText1,
-          ),
-        ],
-      ),
-      const SizedBox(height: 12),
-    ]);
+    return Padding(
+      padding: padding,
+      child: Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+        Text(
+          "preferences".tr(),
+          style: theme.textTheme.ppMori400Black16,
+        ),
+        const SizedBox(
+          height: 24,
+        ),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Text('hide_from_collection'.tr(),
+                    style: theme.textTheme.ppMori400Black16),
+                FlutterSwitch(
+                  height: 25,
+                  width: 48,
+                  toggleSize: 19.2,
+                  padding: 2,
+                  value: isHideGalleryEnabled,
+                  onToggle: (value) async {
+                    await injector<AccountService>()
+                        .setHidePersonaInGallery(widget.persona.uuid, value);
+                    final hiddenAddress =
+                        await injector<AccountService>().getHiddenAddresses();
+                    setState(() {
```

Comment:
safer to add `if (!mounted) return;` before calling setState after calling await.